### PR TITLE
All the tests created pass.

### PR DESCRIPTION
**Pull request by AbdelIdir**
#### What does this PR do?
Choose one or more:

- Adding tests / ensuring code quality

#### Description of Task to be completed? (Copy + Paste Trello card info + link)
Task Description:Creating tests to check the UI for this component, as well as test if the UI change while events are created or not.
Trello Card Link: https://trello.com/c/0nImOlJr/226-front-endtests-eventonboarding

#### How has this been tested & how should it be tested in QA?
- The following tests are in place:
    - UI
    - Events
    - Snapshot
- QA should check:
 -Are the tests relevant.
 -Do the tests match what is on the UI.

#### Any background context you want to provide?

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes

#### Screenshots (if appropriate)
EventOnBoarding

### CODE REVIEW

**Comment by AbdelIdir** [comment]
[Front-End[Tests] => EventOnBoarding](https://trello.com/c/0nImOlJr/226-front-endtests-eventonboarding)

File: src/components/organisms/UserHeader.js [hunk 1]
```diff
@@ -11,8 +11,14 @@ import { Dropdown } from "../atoms/DropDown";
...
   const { email: user } = useSelector(state => state.currentUser);
+  console.log(user);
+
   const initial = user[0].toUpperCase();
 
+  console.log(user[0]);
+
+  debugger;
+
   const Navigation = styled.div`
```
Comment (ELAndrews) [inline src/components/organisms/UserHeader.js]: could we remove the console.log() and debugger

File: src/components/templates/EventOnboarding.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ const BodyContainerColumn = styled(BodyContainer)`
 const EventOnboarding = () => {
   const events = useSelector(state => state.events.data);
   const { userId } = useSelector(state => state.currentUser);
+  console.log(userId);
   const userEvents = events.filter(event => event.creator_id === userId);
```
Comment (ELAndrews) [inline src/components/templates/EventOnboarding.js]: Can we remove this one as well?

File: src/components/templates/__tests__/EventOnboarding.test.js [hunk 3]
```diff
@@ -1,20 +1,98 @@
...
+  it("should not be displaying the 'you have not created events phrase,since two events are already listed' ", () => {
+    let sentence = () =>
+      jestFeatures.queryByText(/You have not created any events yet/);
+
+    expect(sentence()).toBeNull();
+  });
+  it("renders without crashing ", () => {
+    expect(jestFeatures).toMatchSnapshot();
+  });
+});
```
Comment (ELAndrews) [inline src/components/templates/__tests__/EventOnboarding.test.js]: This looks so beautiful!

File: src/components/templates/EventOnboarding.js (hunk 2 above)
Comment (AbdelIdir) [reply src/components/templates/EventOnboarding.js]: Done

**Pull request merged**